File: PHP - CalculatorSkeleton/src/CalculatorBundle/Entity/Calculator.php
<?php
namespace CalculatorBundle\Entity;

class Calculator
{
    /**
     * @var float
     */
    private $leftOperand;

    /**
     * @var float
     */
    private $rightOperand;

    /**
     * @var string
     */
    private $operator;

    /**
     * Get left operand
     *
     * @return float
     */
    public function getLeftOperand()
    {
        return $this->leftOperand;
    }

    /**
     * Set left operand
     *
     * @param float $operand
     *
     * @return float
     */
    public function setLeftOperand($operand)
    {
        return $this->leftOperand = $operand;

        return $this;
    }

    /**
     * Get right operand
     *
     * @return float
     */
    public function getRightOperand()
    {
        return $this->rightOperand;
    }

    /**
     * Set right operand
     *
     * @param float $operand
     *
     * @return float
     */
    public function setRightOperand($operand)
    {
        return $this->rightOperand = $operand;

        return $this;
    }

    /**
     * Get operator
     *
     * @return float
     */
    public function getOperator()
    {
        return $this->operator;
    }

    /**
     * Set operator
     *
     * @param float $operator
     *
     * @return float
     */
    public function setOperator($operator)
    {
        return $this->operator = $operator;

        return $this;
    }
}

File: Java Calculator/src/main/java/com/softuni/Entity/Calculator.java
package com.softuni.Entity;

public class Calculator {
    private Double leftOperand;
    private Double rightOperand;
    private String operator;

    public Calculator() {
    }

    public double getLeftOperand() {
        return leftOperand;
    }

    public void setLeftOperand(double leftOperand) {
        this.leftOperand = leftOperand;
    }

    public double getRightOperand() {
        return rightOperand;
    }

    public void setRightOperand(double rightOperand) {
        this.rightOperand = rightOperand;
    }

    public String getOperator() {
        return operator;
    }

    public void setOperator(String operator) {
        this.operator = operator;
    }

    public Calculator(Double leftOperand, Double rightOperand, String operator) {
        this.leftOperand = leftOperand;
        this.rightOperand = rightOperand;
        this.operator = operator;
    }

    public Double calculateResult(){

        switch (this.operator){
            case "+":
                return this.leftOperand + this.rightOperand;
            case "-":
                return this.leftOperand - this.rightOperand;
            case "*":
                return this.leftOperand * this.rightOperand;
            case "/":
                return this.leftOperand / this.rightOperand;
                default:
                    return null;
        }
    }
}
